bridges/snowbridge/pallets/inbound-queue/src/lib.rs

```diff
@@ -141,6 +141,9 @@ pub mod pallet {
 
 		/// To withdraw and deposit an asset.
 		type AssetTransactor: TransactAsset;
+
+		/// The most expensive xcm here only used to estimate send cost
+		type MaxSendCostXcm: Get<Xcm<()>>;
```

Comment:
Instead of configuring an XCM here to set the fee, I would prefer we use a constant fee and then in a test assert that the const equates to the xcm so we know if when the fee diviates.

Reply:
https://github.com/Snowfork/polkadot-sdk/pull/142/commits/8ad87ba8f861f8020752d52063b017423c80553f